modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/configuration/VulcanJSONConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.vulcan.internal.configuration;
+
+import aQute.bnd.annotation.metatype.Meta;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+
+/**
+ * @author Alejandro Tardín
+ */
+@ExtendedObjectClassDefinition(
+	generateUI = false, scope = ExtendedObjectClassDefinition.Scope.SYSTEM
+)
+@Meta.OCD(
+	id = "com.liferay.portal.vulcan.internal.configuration.VulcanJSONConfiguration"
+)
+public interface VulcanJSONConfiguration {
```

Comment:
Not sure if I like (and if Brian will like) another configuration for Vulcan.

I am also thinking about if we should try to do it in a company scoped way :thinking: but I don't think so

Reply:
Company scoped adds unnecessary complexity. I'll think about a better name maybe.